content/en/docs/ibm/existing-cluster.md

```diff
@@ -4,15 +4,11 @@ description = "Set up a cluster if you already have one"
 weight = 5
                     
 +++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
 
 ## Initial Setup for Existing Cluster
```

Comment:
@adrian555 Thank you for the update to help keep KF docs up to date for v1.1. 

Since it's a small change, can you also change this section title to lower case?

```suggestion
## Initial setup for your existing cluster
```